js/controller.js

```diff
@@ -643,11 +674,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onBuffering: function(event) {
-      if (this.state.isInitialPlay == false && this.state.screenToShow == CONSTANTS.SCREEN.START_SCREEN) {
-        this.state.buffering = false;
-      } else {
-        this.state.buffering = true;
-      }
+      this.state.buffering = !(this.state.isInitialPlay == false && this.state.screenToShow == CONSTANTS.SCREEN.START_SCREEN);
```

Comment:
I'm not sure why this is not showing a conflict. This code became out of date after https://github.com/ooyala/html5-skin/pull/825. The value of `this.state.buffering` is no longer set in this function. Please make sure to update your branch with the latest code from master

Reply:
Actually never mind, I see the conflict now. We generally favor readability, so I recommend just leaving this code as is on the latest version from master.